{
    "question": "Analyze the following piece of Python code: \n```python\nclass NegativeNumberError(Exception):\n    def __init__(self, value):\n        self.value = value\n    def __str__(self):\n        return repr(self.value)\n\ntry:\n    num = int(input(\"Please enter a number: \"))\n    if num < 0:\n        raise NegativeNumberError(\"Invalid entry - negative numbers not permitted\")\nexcept NegativeNumberError as e:\n    print(e.value)\n```\nIf a user inputs -5, which of the following output will you expect to see?",
    "choices": [
        {
            "choice": "Please enter a number: -5\nInvalid entry - negative numbers not permitted",
            "correct": "true"
        },
        {
            "choice": "-5",
            "correct": "false"
        },
        {
            "choice": "NegativeNumberError has been raised",
            "correct": "false"
        }
    ]
}